### PR TITLE
fix: add explicit Profunctor class name

### PR DESCRIPTION
**Pull request by alexlafroscia**
Fixes microstates/ember#98

### CODE REVIEW

File: src/meta.js [hunk 1]
```diff
@@ -62,7 +62,9 @@ export function mount(microstate, substate, key) {
   }, substate);
 }
 
-export const Profunctor = type(class Profunctor {
+export const Profunctor = type(class {
```
Comment (alexlafroscia) [inline src/meta.js]: I decided to remove the class name from here, too, since the minifier will remove it anyway.

**Comment by alexlafroscia** [comment]
@cowboyd @taras any chance you could publish `0.12.4`? I'd love to get this fix, so I don't have to muck with my minifier config anymore!

**Comment by taras** [comment]
I'll do it in 15 minutes.

**Comment by taras** [comment]
@alexlafroscia 0.12.4 is published

**Pull request merged**